build-system/app-index/index.js

```diff
@@ -22,6 +22,8 @@ const fs = BBPromise.promisifyAll(require('fs'));
 const {join, normalize, sep} = require('path');
 const {renderTemplate} = require('./template');
 
+const pc = process;
```

Comment:
Can we just use the full process? I feel like this may be more confusing than helpful? 😮

Reply:
We can, but I named it so for consistency with `app.js`.

Reply:
Hmmm, I'd like to use the standard `process` in both places then, but I guess that would be out of the scope of this PR. 🙃 

I'll approve in favor of consistency as I definitely agree there, and maybe at some point we can change it back? 😄